scss/tech.scss

```diff
@@ -0,0 +1,57 @@
+
+.techs-container {
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.tech {
+  align-items:center;
+  background-color: $green;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  height: 100%;
+  margin: 20px;
+  padding: 10px 30px;
+  position: relative;
+  text-align: center;
+  width: $card-size;
+  z-index: 1;
+}
+
+.tech::before {
+  content: "";
+  bottom: 10px;
+  border: 1px solid $green;
+  border-radius: 4px;
+  left: -10px;
+  position: absolute;
+  right: 10px;
+  top: -10px;
+  z-index: -1;
+}
+
+.tech .tech-img {
```

Comment:
There's no need to use both classes since you are not overriding anything. It's better if you stay simple with only one class (`.tech-img`) in this case, so if the moment comes, you can override the styles easily :smile_cat: